### DISCUSSION TITLE
Can't run on Apple M1 Pro - error from yarn

### DISCUSSION THREAD

**Discussion by LiveRock** [streamlink/streamlink-twitch-gui#953] [Q&A]
Hi

After the following:
```
git clone https://github.com/streamlink/streamlink-twitch-gui.git
cd streamlink-twitch-gui
yarn install
yarn run grunt release
```

The last line I get these errors:

```
Running "release" task
Warning: Unsupported platform: darwin (arm64) Use --force to continue.

Aborted due to warnings.
error Command failed with exit code 3.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

I used --force but still unable to run.

What did I do wrong here?

**Comment by bastimeyer** [comment]
> Apple M1
> Unsupported platform: darwin (arm64)

Only `x64` is supported. Your NodeJS reports `arm64`.
https://nodejs.org/api/process.html#processarch

**Comment by adriel** [comment]
Also, for anyone else coming here, wanting to just run the app on an M1 / M2, you can just run `brew install --cask streamlink-twitch-gui` and it'll install the intel version, which works fine via Rosetta (obviously need to have Homebrew installed).

Looks like a native build is in the works though (which I'm pretty excited for): https://github.com/streamlink/streamlink-twitch-gui/issues/969
